addons/mail/static/src/core/common/chat_window.js

```diff
@@ -91,7 +91,7 @@ export class ChatWindow extends Component {
             ev.stopPropagation();
             return;
         }
-        if (ev.target.closest(".o-dropdown")) {
+        if (ev.target.closest(".o-dropdown") || ev.target.closest(".o-mail-AutoresizeInput")) {
             return;
```

Comment:
I suggest to switch focus to `composer` here. As it happens with thread-acitons and in Discuss aswell.